cafebabel/templates/articles/tags/detail.html

```diff
@@ -1,20 +1,29 @@
 {% extends "base.html" %}
 
+{% block extra_js %}<script src={{ url_for('static', filename='js/tag-parallax.js') }}></script>{% endblock %}
+
 {% block meta_title %}{{ tag.name }}{% endblock %}
 {% block meta_description %}{{ tag.summary }}{% endblock %}
 
-{% block body_class %}article-tag{% endblock %}
+{% block body_class %}tag-page{% endblock %}
 {% block body %}
   {% if tag.has_image %}
-    <img src="{{ tag.image_url }}" alt="Image for {{ tag.name }}">
+    <div class=tag-header-image style="background:url({{ tag.image_url }}) no-repeat center 0 fixed"></div>
```

Comment:
I'm strongly against that change :)

My take is: do whatever you want with JS but keep at least native HTML elements please.

Reply:
ok, @pointbar va m'expliquer comment faire via js pas de souci 👌